docs/migration_guide.rst

```diff
@@ -8,7 +8,18 @@ Migration guide
 Breaking changes
 ----------------
 
-- :meth:`~starknet_py.hash.utils.pedersen_hash` function implementation has been changed, now uses only `crypto-cpp-py` library implementation.
+.. currentmodule:: starknet_py.net.account.base_account
```

Comment:
I'm not sure if we should use it here. This could lead to hard-to-find problems in the future. 
wdyt @cptartur?

Reply:
Migration guide file is written from the bottom-up, so I think there shouldn't be any problems.

Reply:
Ok, fair, but what are the advantages of this approach?

Reply:
BaseAccount.verify_message is linked to api section

Reply:
And we could do `starknet_py.net.account.base_account.BaseAccount.verify_message` as always

Reply:
Then it's either the long path with all the package names you listed or just `verify_message` if you precede the path with a tilde `~`.

Current solution leaves only `BaseAccount.verify_message` with a reference.